paddle/phi/api/yaml/sparse_ops.yaml

```diff
@@ -489,3 +489,14 @@
     func : sync_batch_norm_coo{sparse_coo, dense, dense, dense, dense -> sparse_coo, dense, dense, dense, dense, dense}
     data_type : x
   backward : sync_batch_norm_grad
+
+- op : reshape
+  args : (Tensor x,  IntArray shape)
+  output : Tensor(out)
+  infer_meta :
+    func : ReshapeInferMeta
+  kernel :
+    func : reshape_coo{sparse_coo -> sparse_coo},
+           reshape_csr{sparse_csr -> sparse_csr}
```

Comment:
the design of `reshape_csr` needs to be added to [rfc](https://github.com/PaddlePaddle/community/blob/master/rfcs/APIs/20220905_api_design_for_sparse_reshape.md)

Reply:
Do you mean I need to modify the RFC to make the content of  the RFC  agree with my actual  implementation ? Ok, I will do it.